src/cmap/options.rs

```diff
@@ -109,18 +99,18 @@ pub struct ConnectionPoolOptions {
 
 impl ConnectionPoolOptions {
     pub(crate) fn from_client_options(options: &ClientOptions) -> Self {
-        Self::builder()
```

Comment:
This situation here is one of the more painful examples not being able to pass in an `Option<T>` anymore, though I doubt this will be common among our users. The [standard examples for the builder pattern](https://doc.rust-lang.org/1.0.0/style/ownership/builders.html) all don't take in `Option`, and it seems neither does the actual [`std::process::Command`](https://doc.rust-lang.org/std/process/struct.Command.html) that it's based off of, so I think we should be okay here.

Reply:
I agree, it doesn't seem like much of an issue. I can't think of a lot of use cases for storing some value you're going to pass in as an option in an optional variable